### DISCUSSION TITLE
Document does not contain information in standalone project - help integrating

### DISCUSSION THREAD

**Discussion by cboldon** [smallrye/smallrye-open-api#1403] [Q&A]
I'm trying to integrate with a standalone RestEasy project. I've tried scanning, passing in specific classes and packages, etc, but the OpenApiDocument remains empty:

openapi: 3.0.3
info:
  title: Generated API
  version: "1.0"
paths: {}

I am integrating as follows (just hacking things together and trying to get things working-ish):
```
OpenApiDocument document = OpenApiDocument.INSTANCE;

Set<String> classSet = new HashSet<>();
classSet.add(MyEndpoint.class.toString());

MyOpenApiConfig openApiConfig = new MyOpenApiConfig(); // (Implements OpenApiConfig just so I can hard code)
openApiConfig.setScanClassStrSet(classSet);

IndexView index = null;
try {
    index = Index.of(MyEndpoint.class);
} catch (IOException e) {
    log.error("Could not create IndexView", e);
}

document.set(OpenApiProcessor.bootstrap(openApiConfig, index));
```

Could someone please point me in the right direction? It would be very much appreciated.

**Comment by MikeEdgar** [comment]
`Class#toString` doesn't give just the class's name.

Replace:
```java
classSet.add(MyEndpoint.class.toString());
```
With:
```java
classSet.add(MyEndpoint.class.getName());
```

**Reply by codespearhead** [reply]
@cboldon It seems that the above suggestion solved the problem. Can you [mark this discussion as answered](https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions#marking-a-comment-as-an-answer)?

**Comment by cboldon** [comment]
Thank you for saving me from my stupidity.
